content/200-concepts/100-components/02-prisma-client/058-transactions.mdx

```diff
@@ -302,6 +302,73 @@ transaction functions. We recommend you get in and out as quick as possible!
 
 </Admonition>
 
+### Transaction timing issues
+
+<Admonition type = "note">
+
+- This section is not available on MongoDB, because MongoDB does not support [isolation levels](https://www.prisma.io/dataguide/intro/database-glossary#isolation-levels).
```

Comment:
```suggestion
- This section does not apply to MongoDB, because MongoDB does not support [isolation levels](https://www.prisma.io/dataguide/intro/database-glossary#isolation-levels).
```